src/SourceBuild/content/repo-projects/Directory.Build.targets

```
@@ -287,6 +287,64 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyRepoArtifacts"
+          AfterTargets="Package"
+          Condition="Exists($(RepoManifestFile))">
```

I think we should use a better condition here as Exists can easily fail if for some reason the file didn't get created, without causing the build to fail.

```suggestion
          Condition="'$(UseInnerClone)' != 'true'">
```

---

Fixed with https://github.com/dotnet/installer/pull/18153/commits/9bfacadeccfd5235e7e99c2d624cb92b7807b176